smartsim/entity/__init__.py

```diff
@@ -3,3 +3,4 @@
 from .entity import SmartSimEntity
 from .entityList import EntityList
 from .model import Model
+from .dbobject import DBScript, DBModel, DBObject
```

Comment:
might be time to define an ``__all__`` here similar to settings when we need to simply import all entity definitions into a file.

Reply:
Done